pkg/server/handler/tikvhandler/tikv_handler.go

```diff
@@ -1995,33 +1984,6 @@ func (h *TestHandler) handleGCResolveLocks(w http.ResponseWriter, req *http.Requ
 	}
 }
 
-// ServeHTTP handles request of resigning ddl owner.
-func (h DDLHookHandler) ServeHTTP(w http.ResponseWriter, req *http.Request) {
```

Comment:
@AilinKid told me that it's used in other tests

Reply:
I found it's https://github.com/PingCAP-QE/automated-tests/blob/44e476617b26efdf344731d2d38393bbd8d76f1c/ticases/column_type_change/ctc_correctness_check/ctc_correctness_check.go#L51 

I will try to use add a failpoint and use `/fail/` TiDB HTTP API to replace it.

Reply:
lgtm, maybe add a common api to enable failpoint dynamically, and better implement it at failpoint repo which i am not sure whether it exists

Reply:
already there

https://github.com/pingcap/tidb/blob/3dfa15cd978624b10ea74931dcf171cc8dc6c8bb/pkg/server/http_status.go#L422-L430